typings.d.ts

```diff
@@ -0,0 +1 @@
+declare module '*';
```

Comment:
Why is this required? The reason I am asking is to know what problem you faced (I know that we need to do something like this to make this work).

Reply:
This is to avoid compiler warning for having declaration files for each and every util we use.